### ISSUE TITLE
Find a way to keep "objectives" folder contents up-to-date (possibly automation)

### ISSUE THREAD

**Issue by vegetabill** [Techtonica/curriculum#1210]
#### we can try embedme, which we used at gap and it seems to work great: https://github.com/zakhenry/embedme.
--------
Original issue:
The objectives files are aggregated from all of the individual lesson files. Michelle said the audience for these was the SEAM and sponsor companies who wanted more details beyond the [general curriculum summary doc](https://docs.google.com/document/d/1Pr44pAKmq0NvfFiFEtqULzHPBZQnNsc6o040Qd4XxiE/edit). (admin only)

IMHO, these files are not very valuable to either audience group due to being overly detailed. Because they are duplicated data, they will also always be out of date and a maintenance burden. I suggest we remove them. If engineering orgs need more details, we should just direct them to the root README which gives detail and links for the contents. Any SEAM will need to build their own mental of the curriculum, which I plan to help by building something more visual/high-level (see #1209 )

Discuss...

**Comment by vegetabill** [comment]
cc @gsong @MichelleGlauser @alodahl for their input

**Comment by gsong** [comment]
I agree that a half sheet with simple visuals would be more effective. Not sure what "general curriculum summary doc" refers to, since the link is blank.

**Comment by vegetabill** [comment]
@gsong sorry fixed the link. It's a google doc someone made: https://docs.google.com/document/d/1Pr44pAKmq0NvfFiFEtqULzHPBZQnNsc6o040Qd4XxiE/edit

**Comment by gsong** [comment]
A suggestion on the document you're creating (and this may already exist), I would also talk about what are some appropriate roles or specific tasks you feel a Techtonica apprentice would be able to perform by the end of the program.

I'm not as interested in what they've learned (other than broad strokes) vs. what can they realistically do for me. Hope that makes sense.

**Comment by alodahl** [comment]
Hi @vegetabill , we discussed this over hangouts, but for the record:

1. This was compiled to be a checklist for graders who are looking at the assessments so they can judge whether or not the objectives were met. If not, graders know which outline to refer the learner to for more study.
2.  This was also compiled to be a checklist to make sure that all of the objectives were actually addressed in the first draft's assessments as they were being written.
3. It was not meant to be a secret from apprentices; these are in the outlines, so we do expect the apprentices to know them. They could be used as a study guide, although they are not really formatted for that purpose.
4. If you're worried about going out of sync, we can try embedme, which we used at gap and it seems to work great: https://github.com/zakhenry/embedme.  Also, this would be a great, fairly simple issue for gssoc people.   However, if you have solutions for #1 and #2 already, we don't need this.
